soc/arm/nuvoton_npcx/common/soc_i2c.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_NPCX_SOC_I2C_H_
+#define _NUVOTON_NPCX_SOC_I2C_H_
+
```

Comment:
Generally headers (and implementation) should include the headers they depend on, so at minimum `<device.h>` should be included here.

Reply:
Done.